frontend/src/main/index.js

```diff
@@ -10,7 +10,7 @@ try {
   if (process.env.ELEC_ENV === 'dev') {
     freePort = 9002;
   } else {
-    freePort = getPortSync();
+    freePort = getPort();
```

Comment:
Doesn't `getPort` return a `Promise<number>`? I'm pretty sure this should be `await`ed.